Sources/Loaf/Loaf.swift

```diff
@@ -174,6 +176,21 @@ final public class Loaf {
         self.sender = sender
     }
     
+    /// This initializer is intended for use with SwiftUI
+    public init(_ message: String,
```

Comment:
We can't pass a `UIViewController` to a Loaf in SwiftUI because they don't exist, so a new constructor is exposed without `sender`, and `LoafView` handles setting up the sender later internally, without exposing details publicly.